content/docs/quickstart.mdx

```diff
@@ -22,74 +22,216 @@ keywords:
+
+In the Zero onboarding screen, select the **Docker** tab and copy the Docker Compose configuration. (If you selected **Finish** before copying the Docker configuration, we've provided a copy below.)
+
+Paste the configuration in a `compose.yaml` file:
+
+```yaml title="compose.yaml" {8,15} showLineNumbers
+services:
+  pomerium:
+    image: pomerium/pomerium:v0.27.0
+    ports:
+      - 443:443
+    restart: always
+    environment:
+      POMERIUM_ZERO_TOKEN: <CLUSTER_TOKEN>
+      XDG_CACHE_HOME: /var/cache
+    volumes:
+      - pomerium-cache:/var/cache
+    networks:
+      main:
+        aliases:
+        - verify.<CLUSTER_STARTER_SUBDOMAIN>.pomerium.app
+  verify:
+    image: cr.pomerium.com/pomerium/verify:latest
+    networks:
+      main:
+        aliases:
+        - verify
+
+networks:
+  main: {}
+
+volumes:
+  pomerium-cache:
+```
 
-# Run Pomerium Core With Docker
+Replace the following values:
 
-Run Pomerium Core with Docker containers in **under 5 minutes**.
+- **Line 8**: Replace `CLUSTER_TOKEN` with the cluster token generated for you in the onboarding config
+- **Line 15**: Replace `CLUSTER_STARTER_SUBDOMAIN` with your generated starter subdomain
 
-The Core quickstart uses Pomerium's [**Hosted Authenticate Service**](/docs/capabilities/hosted-authenticate-service), but you can also configure a [**self-hosted authenticate service**](/docs/capabilities/self-hosted-authenticate-service) to integrate with Pomerium.
+:::info Where's my **Starter Domain?**
 
-## Prerequisites
+You can find your **Starter Domain** at the top of the Zero Console in the navigation bar.
 
-[Docker] and [Docker Compose]
+:::
 
-## Configure Pomerium
 
-Create a `config.yaml` file in the root of your project.
+## Connect to Pomerium Zero
 
-Add the configuration below to `config.yaml`:
+Next, deploy Pomerium Zero with the following command:
 
-<ConfigDocker />
+```bash
+docker compose up -d
+```
 
-## Configure Docker
+This command will deploy a local cluster running Pomerium in a Docker container. If your cluster deploys successfully, the Pomerium Zero cloud will attempt to secure a connection to your cluster.
 
-Create a `docker-compose.yaml` file in the root of your project.
+If Pomerium Zero connects successfully, it will notify you in the onboarding screen.
 
-Add the configuration below to `docker-compose.yaml`:
+![The successful connection notification in the Pomerium Zero onboarding flow](./img/quickstart/zero-quickstart-connection.png)
 
-<DockerCompose />
+Once you've connected, select **Finish**.
 
-## Run Docker Compose
+:::info
+
+See the [Clusters Concept] page to learn more about clusters and how they work with the Pomerium Zero cloud.
+
+:::
+
+## Connect to upstream services
+
+In the Zero Console, we've provisioned two starter routes, **SSH** and **Verify**, that come with their own DNS records and TLS certificates:
+
+![Viewing the two starter routes](./img/quickstart/pz-quickstart-routes.png)
+
+### Connect to SSH
```

Comment:
I'd recommend starting with the simpler example service first (i.e. swapping this section and the "Connect to Verify" section).

Reply:
Agree.